### DISCUSSION TITLE
Wanna know a resource to connect our fundme project with react js any resource out there.

### DISCUSSION THREAD

**Discussion by Aashim-Limbu** [Cyfrin/foundry-full-course-cu#2880] [Q&A]
i want to integrate Foundry Contract with Nextjs / React.js is there a better resouce that would guide me. At this time i even don't know if i should write the integration logic in client side or the server side. On research i've found that metamask inject some window.ethereum in client side so i assume it should be implemented in client side only. But i need a resource . Can anyone help me ?

**Comment by EngrPips** [comment]
I don't know any explicit resources, but I assume you will get many of them with a Google or YouTube search. How do you mean `At this time i even don't know if i should write the integration logic in client side or the server side`

**Reply by Aashim-Limbu** [reply]
oh i mean if we could write the the logic in our backend/[server code that runs on server not on client] to connect with the wallet .😀

**Reply by Aashim-Limbu** [reply]
well i have found some resource out there . 😀

**Reply by EngrPips** [reply]
Isn't the blockchain your backend in dapp development?

**Reply by EngrPips** [reply]
> well i have found some resource out there . 😀

Resources are always lying everywhere for whoever wants it
